pkg/config/service.go

```diff
@@ -39,6 +41,7 @@ func GetServiceConfig() *ServiceConfig {
 		S3Prefix:           os.Getenv(s3PrefixEnvVar),
 		ClusterName:        os.Getenv(clusterNameEnvVar),
 		UsePathStyle:       os.Getenv(usePathStyleEnvVar) == "false",
+		DisableEncryption:  os.Getenv(disableEncryptionEnvVar) == "true",
```

Comment:
changed this to true to be explicity always enabled

Reply:
I also noticed that since the bump deps it is now Go 1.23 and I have a `[info] /home/cedric/sdk/go1.23.0/src/slices/iter.go:50:17 cannot range over seq (variable of type iter.Seq[E]) (compile)` that I didn't have before, not sure why since it does not seem related to what I have changed but still, might be useful to look into it